man/tpm2_unseal.8.in

```diff
@@ -52,6 +52,9 @@ Output file name, containing the unsealed data. Defaults to stdout if not specif
 .TP
 \fB\-X ,\-\-passwdInHex\fR
 passwords given by any options are hex format.
+.TP
+\fB\-S ,\-\-input-session-handle\fR
+Optional Input session handle from a policy session for authorization. [Only for device-tcti]
```

Comment:
This isn't even true 100% now. Now that Kernel 4.12 has an in-kernel resource manager, using device tcit on the /dev/tpmrm0 node will cause issues when the command exits and a flush is issued.

But, if something like tpm-shell is introduced, then resource managers will not be a problem.

So, do we really want to annotate this here? I would say just drop it.